src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -352,6 +351,9 @@ public Builder runDevAppServerWait(int runDevAppServerWaitSeconds) {
 
     /**
      * Create a new instance of {@link CloudSdk}.
+     *
```

Comment:
I think you need `<p>` tag here. See: https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs